.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "\U00002753Questions/Help/Support"
```

Comment:
Another really small detail but I personally find it a bit confusing that "Support" is included here in the name while it sounds like you're being referred to SO etc. when asking for it. Are they referring to different "support"s?

Reply:
Honestly, I just copied and pasted the title from [Pytorch's issue template title](https://github.com/pytorch/pytorch/issues/new/choose), so there is no important meaning about the word "support".
I think that we might be able to remove "support" from the title.

Reply:
Or just simply `Questions` 🙂 . If we change the body however, I think any of them will do.

Reply:
> just simply `Questions`

Nice idea 👍 
Fixed: https://github.com/pfnet/optuna/pull/652/commits/e01aefb2340f74dc2ba6963b6d9f5347a1099b3f

Reply:
In addition, I updated the body of this template (see: https://github.com/pfnet/optuna/pull/652/commits/51a38b89a692001655c9328e95ef91ffb423c451).

Reply:
Awesome, I'll take a look!